code/game/objects/structures/morgue.dm

```diff
@@ -241,15 +241,10 @@ GLOBAL_LIST_EMPTY(crematoriums)
 				M.ghostize()
 				qdel(M)
 
-		var/ash_check = FALSE
 		for(var/obj/O in conts) //conts defined above, ignores crematorium and tray
-			if(istype(O,/obj/effect/decal/cleanable/ash)) ash_check = TRUE//creates the illusion of ash piling up
 			qdel(O)
 
-		var/obj/effect/decal/cleanable/ash/a
-		if(ash_check) a=new/obj/effect/decal/cleanable/ash/large(src)//cont. illusion of more ash
-		else a=new/obj/effect/decal/cleanable/ash(src)
-		a.layer = connected.layer//Makes the ash the same layer as the tray.
+		new/obj/effect/decal/cleanable/ash(src)

```

Comment:
You can probably revert your changes here and check the review on `cleanable/misc.dm` for one way to get this to work fairly cleanly.